### PR TITLE
Ansible Module: ssh_key_info

### PR DESCRIPTION
**Pull request by zliang-akamai**
Add an Ansible module, `ssh_key_info`, and its tests and docs.

### CODE REVIEW

**Review by jriddle-linode — APPROVED** [review]

File: tests/integration/targets/ssh_key_info/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        r: "{{ 1000000000 | random }}"
+
+    - linode.cloud.api_request:
+        api_token: '{{ api_token }}'
+        ua_prefix: '{{ ua_prefix }}'
+
+        path: profile/sshkeys
+        method: POST
+        body:
+          ssh_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDDOUaSPY2foeZuDmaIV6bmYnwMVcVdyhA3fF7y/Jg44tEfwEmH7DC+ixwvnyBgHhBCaQsv4S97sphFaC0R56CBwVlVC//sF/E9IpuiCkxjfFbQJlISa4moeH8CfGCgNyr97fPxhP0vJWeoqFaBRCDu8avdaX5VJ/QDN6J8faf5nQ32OHrO6/vQAVyceUwYnZTJFECc7okdbVl3byygTyE/+7O937ljFeMSd84C4wMGXyxks9efuZVZofnw9PCN6BWRlBqBj/nmUXfMcTHRU8Vw4EK+A0+Rz7LTEVv2z0h+M0R42lURJT3B5DLpuz5ZAbS2Vu1dYLjLpqnkJ209l6rVFKb/Z/bycN/f9DPNPd8wqy14bgk3SQw1aHE83ADctCMZYYhlL93Yo8COdxaWfLco/+gfOTn/Hyn8vaG3pr/VmP6lFh7Hv61+idR5t27N89kREW+vBLDvGd+kZWTHOrTu+VySKHiIWjj7/bbfBoTQr0Yq9y8r9mRf5pTaz9rrY50= zliang@dfw-mpzpg"
```
Comment (lgarber-akamai) [inline tests/integration/targets/ssh_key_info/tasks/main.yaml:13]: Just double checking that this key is only used for this test

**Review by lgarber-akamai — APPROVED** [review]
Looks good! Could you run `make gendocs` and push up the changes before merging?

File: tests/integration/targets/ssh_key_info/tasks/main.yaml (hunk 1 above)
Comment (zliang-akamai) [reply tests/integration/targets/ssh_key_info/tasks/main.yaml:13]: Oh, let me fix it

Comment (zliang-akamai) [reply tests/integration/targets/ssh_key_info/tasks/main.yaml]: I moved it to fact section. And yes, it was only use for this test.

**Pull request merged**